.github/workflows/scheduled.yaml

```diff
@@ -0,0 +1,24 @@
+name: Property Testings
+
+on:
+  schedule:
+    - cron: '0 */4 * * *'
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  property-testing:
+    name: Property Testing
+    runs-on: ubuntu-latest
+    env:
+      PROPTEST_CASE: 1000
```

Comment:
1000 is insane, I would recommend 100. 
Especially since 100 of each tests already takes between 30min and 1h, I think 1000 would take way more than 4h.

Reply:
Ok... let's reduce that to 100. Done.